ops/fast_call.rs

```diff
@@ -250,13 +250,19 @@ pub(crate) fn generate(
       //
       // V8 calls the slow path so we can take the slot
       // value and throw.
-      let result_wrap = q!(Vars { op_state }, {
+      let default = if optimizer.fast_result == Some(FastValue::Pointer) {
+        // Pointers do not have default value.
+        q!({ ::std::ptr::null_mut() })
+      } else {
+        q!({ Default::default() })
+      };
```

Comment:
nit: this logic should be a method `FastValue::default_value()`.
```suggestion
      let default = optimizer.fast_result.default_value();
```